test/typecheck.jl

```diff
@@ -148,7 +148,7 @@ end
 msgs = lintstr(s)
 @test msgs[1].code == :E521
 @test msgs[1].variable == "a"
-@test contains(msgs[1].message, "apparent type DataType is not a container type")
+@test contains(msgs[1].message, "apparent type Type")
```

Comment:
I think, going forward, we will want to standardize the error messages based off error codes, and then disassociate the testing of linting creating the right error code, and testing the error codes being associated with an appropriately descriptive message.